_posts/2021-11-02-6.0.0-rs-release.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title: Apache Arrow Rust 6.0.0 Release
+date: "2021-11-08 00:00:00"
+author: pmc and contributors
```

Comment:
I don't think this works right; either just say 'pmc', or a single person named in https://github.com/apache/arrow-site/blob/master/_data/contributors.yml (possibly adding yourself there), or plain text. (As it currently stands, this would literally print "pmc and contributors" as the author, and I don't think we want this.)

Reply:
@nealrichardson Leave as PMC. Will add myself to contributors later. So if ok, please merge.

Reply:
May want to have something in addition to PMC, since a lot of this is/can be automated so project newcomers and committers can also help with this. Maybe there should be PC - Project Contributors? For posts on releases, the role is more of editing and curating rather than authoring despite the suggestion at https://github.com/apache/arrow-site/blob/master/README.md